readme.md

```diff
@@ -8,28 +8,47 @@
 
 ## Overview
 
+Chief-of-state (COS) is a clustered persistence framework for building event sourced applications.
+COS supports CQRS and event-sourcing through simple, language-agnostic interfaces via gRPC, and it
+allows developers to describe their schema with Protobuf. Under the hood, COS leverages [Akka](https://akka.io/)
+to scale out and guarantee performant, reliable persistence.
+
+Chief-of-state was built by Namely with the following principles:
+* Wire format should be the same as persistence
+* Scaling should not require re-architecture
+* Developers shouldn't face race conditions or database locks
+* Rules should be enforced with interfaces
+* Event sourcing is a valuable, but challenging to implement
```

Comment:
* Event sourcing is ~a~ valuable, but challenging to implement

Reply:
haha